ruby/vampires.rb

```diff
@@ -1,4 +1,8 @@
+
+def vampire
+
 current_year = 2017
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.